sauce/features/accounts/customFlagNames/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'CustomFlagNames',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Add Custom Flag Names',
```

Comment:
Changes have been made to the title and description. Hopefully that is more descriptive and helpful. Thanks for the input.